android/lib/src/main/java/com/ern/api/impl/core/ElectrodeBaseFragmentDelegate.java

```diff
@@ -27,7 +27,7 @@
 
 public class ElectrodeBaseFragmentDelegate<T extends ElectrodeBaseFragmentDelegate.ElectrodeActivityListener, C extends ElectrodeFragmentConfig> implements LifecycleObserver {
     private static final String TAG = ElectrodeBaseFragmentDelegate.class.getSimpleName();
-    protected static final String KEY_UNIQUE_VIEW_IDENTIFIER = "viewId";
+    public static final String KEY_UNIQUE_VIEW_IDENTIFIER = "viewId";
```

Comment:
I assume the visibility change here is only because of tests? Have you considered a different approach (e.g. using package-private and placing the test in the same package). Not sure if it makes sense here, so just checking..

Reply:
This `KEY_UNIQUE_VIEW_IDENTIFIER` is inside `core` and the test would be inside `navigation` package, so package-private won't work in this case.